packages/shared-components/package.json

```diff
@@ -36,22 +36,28 @@
     "build:watch": "stencil build --watch",
     "build": "stencil build",
     "start": "stencil build --dev --watch --serve",
-    "test": "stencil test --spec --e2e",
-    "test:watch": "stencil test --spec --e2e --watchAll",
-    "test:spec:watch": "stencil test --spec --watchAll",
-    "test:e2e:watch": "stencil test --e2e --watchAll",
```

Comment:
these last two commands you deleted are used to run the tests separately, so better restore them

Reply:
done